index.bs

```diff
@@ -44,6 +44,105 @@ url: https://dom.spec.whatwg.org/#ref-for-concept-getelementsbytagname; type: in
     }
 </style>
 
+<h2 id="api-principles">Principles behind design of Web APIs</h2>
+
+<h3 id="safe-to-browse">It should be safe to visit a web page</h3>
+
+Hyperlinks, links from one page to another, are one of the foundations of the Web.
+
+Following a link, or visiting a web page, should be safe.
+Users who are following a link to a web page they haven't seen before
+should not have to fear for the security of their computer
+or for essential aspects of their privacy.
+(One caveat here is that users of the Web
+should not assume that they are free from consequences
+if their use of the Web is harming others.)
+
+Saying “essential aspects” here skips over quite a bit of detail.
+The Web isn't perfectly private.
+But we can probably make it better than it is today.
+In doing so, we may want to focus on the cases where
+user expectations of what information is revealed when using the Web
+don't match the technical details.
+We might be able to use the design of user interfaces
+to make it clear what users of the Web should expect,
+since one aspect of privacy problems is when reality doesn't meet expectations.
+For example,
+a person walking down the street generally expects to be recognized by their friends,
+but depending on the country,
+may not expect that they were walking down that street to be recorded in a permanent government database.
```

Comment:
this sentence reads strangely to me. I think I know what you're trying to say, but this is a confusing way to say it.

Does this read better to anyone else:

> For example, a person walking down the street generally expects to be recognized by their friends but,
> depending on the country,
> they may not expect the fact that they walked down that particular street at that particular time to be recorded in a permanent government database.

Reply:
I agree it's better.  I took most of your suggestions but turned "depending on the country" into a parenthetical and omitted a few words ("the fact", "particular", "particular").